OpenRA.Mods.Common/Traits/BodyOrientation.cs

```diff
@@ -76,9 +76,14 @@ public BodyOrientation(ActorInitializer init, BodyOrientationInfo info)
 				var isb = self.HasTrait<ISpriteBody>();
```

Comment:
Looks like you can move this after the ```qboi == null``` check and avoid the lookup most of the time.